static/app/components/events/featureFlags/onboardingIntegrationSection.tsx

```diff
@@ -86,7 +86,7 @@ export default function OnboardingIntegrationSection({
   }
 
   return (
-    <Access access={['org:write']}>
+    <Access access={['org:read']}>
```

Comment:
this just looks wrong, we're doing a write operation. 

are we sure that resource permissions aren't the problem?

Reply:
https://github.com/getsentry/sentry/blob/56fb94a64f1080bdb1d46ed117f6ee7ad3ba5b61/src/sentry/api/bases/organization.py#L215-L221

Reply:
Its the established norm.  Probably the Access wrapper needs to be removed?

Reply:
does this mean that a user would need all three permissions in order to do a post?
`"POST": ["org:read", "org:write", "org:admin"],`

because the javascript above is saying that the user only needs `'org:read'` to interact with the form, and presumably if they don't have the other two permissions the api endpoint would barf back at them.

Reply:
Any of the three are required to post.

Reply:
that's surprising

but also, the ui only looks at the one not the other two :(

Reply:
Should I remove the wrapping <Access> tag?

Reply:
You could do like:
```javascript
import {hasEveryAccess} from 'sentry/components/acl/access';

const canRead = hasEveryAccess('org:read', {organization});
const canWrite = hasEveryAccess('org:write', {organization});
const canAdmin = hasEveryAccess('org:admin', {organization});
const hasAccess = canRead || canWrite || canAdmin;
```
and remove the `<Access>` wrapper

Reply:
Rewrote it.  The indentation change makes the diff look huge 🤷

Reply:
github is here for us:
![SCR-20241205-jsey](https://github.com/user-attachments/assets/58bdea62-0418-4d99-915f-7f8d3e06ec95)